package java_fundamentals.class_problems;
import java.util.Scanner;
public class Q7_FactorialCalculation {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        long factorial = 1;
        for (int i = 2; i <= n; i++) factorial *= i;
        System.out.println(factorial);
    }
}